docs/v1.X/docs/bouncers/index.md

```diff
@@ -8,5 +8,18 @@ They can either be within the applicative stack, or work out of band :
 [nginx bouncer](https://github.com/crowdsecurity/cs-nginx-bouncer) will check every unknown IP against the database before letting go through or serving a *403* to the user, while a [firewall bouncer](https://github.com/crowdsecurity/cs-firewall-bouncer) will simply "add" malevolent IPs to nftables/ipset set of blacklisted IPs.
 
 
-You can explore [available {{v1X.bouncers.name}} on the hub]({{v1X.hub.bouncers_url}}), and find below a few of the "main" {{v1X.bouncers.name}} :
+You can explore [available {{v1X.bouncers.name}} on the hub]({{v1X.hub.bouncers_url}}).
 
+
+To be able for your {{v1X.bouncers.Name}} to communicate with the local API, you have to generate an API token with `cscli` and put it in your {{v1X.bouncers.Name}} configuration file:
```

Comment:
if we have a parameter that is generally used - i think it's the case - maybe we should document it ?